app/Controllers/LanguagePackageController.js

```diff
@@ -39,7 +43,12 @@ async function sendLanguagePackages(req, res) {
 
   await Promise.all(
     Object.keys(languagePackages).map(async (key) => {
-      languagePackages[key].dataValues.unresolvedQuerys = await getUnresolvedVocabulary(
+      languagePackages[key].dataValues.unresolvedQuerys = await getNumberOfUnresolvedVocabulary(
+        languagePackages[key].id,
+        userId
+      );
+      // add number of unactivated vocabularies
+      languagePackages[key].dataValues.unactivatedVocabularies = await getNumberOfUnactivatedVocabulary(
```

Comment:
I think overriding the `dataValues` prop isn't a good solution. I would do it this way.
```js
async function sendLanguagePackages(req, res) {
  // get userId from request
  const { id } = req.user;

  // create language Package
  const languagePackages = await getLanguagePackages(id, res).map((languagePackage) => languagePackage.toJSON());

  const formatted = await Promise.all(
    languagePackages.map(async (languagePackage) => ({
      unresolvedQueries: await getNumberOfUnresolvedVocabulary(languagePackage.id, id),

      // add number of unactivated vocabularies
      unactivatedVocabularies: await getNumberOfUnactivatedVocabulary(languagePackage.id, id),

      ...languagePackage,
    }))
  );

  res.send(formatted);
}
```